### DISCUSSION TITLE
能够在WINDOWS下进行编译么该版本包？

### DISCUSSION THREAD

**Discussion by frankcgq105** [timonwong/prometheus-webhook-dingtalk#156] [Q&A] [answered]
OS SYSTEM：WIN 7
不知道是否支持在WIN下进行编译？
尝试在prometheus-webhook-dingtalk\cmd\prometheus-webhook-dingtalk文件夹里面
输入命令go build main.go
但是提示如下：
go: github.com/Masterminds/sprig/v3@v3.0.0: Get "https://proxy.golang.org/github
.com/%21masterminds/sprig/v3/@v/v3.0.0.mod": dial tcp 172.217.160.81:443: connec
tex: A connection attempt failed because the connected party did not properly re
spond after a period of time, or established connection failed because connected
 host has failed to respond.
不知道是否也有人在WIN下或者LINUX下进行过编译？

**Comment by yabowang** [comment]
我在Ubuntu下编译了，搞了一天，需要安装nodejs + go + yarn

**Comment by timonwong** [comment, marked as answer]
@frankcgq105 编译需要 nodejs, go, yarn，如 @yabowang 所说

你这里的错误主要在于国内的由于众所周知的原因连接 Google 会很不「稳定」，建议你使用一个国内的 go mod proxy，比如:
- https://goproxy.cn/
- https://goproxy.io/
